docs/add-ins/get-and-set-recurrence.md

```diff
@@ -0,0 +1,190 @@
+
+**Table 2. Recurrence types and their applicable properties**
+
+|[Recurrence type](/javascript/api/outlook_1_7/office.mailboxenums.recurrencetype)|Valid [recurrence properties](/javascript/api/outlook_1_7/office.recurrenceproperties)|Usage|
+|---|---|---|
+|`daily`|- [`interval`][interval link]|An appointment occurs every *interval* days. Example: An appointment occurs every **_2_** days.|
+|`weekday`||An appointment occurs every weekday.|
+|`monthly`|- [`interval`][interval link]<br/>- [`dayOfMonth`][dayOfMonth link]<br/>- [`dayOfWeek`][dayOfWeek link]<br/>- [`weekNumber`][weekNumber link]|1. An appointment occurs on day *dayOfMonth* every *interval* months. Example: An appointment occurs on day **_5_** every **_4_** months.<br/>2. An appointment occurs on the *weekNumber* *dayOfWeek* every *interval* months. Example: An appointment occurs on the **_third_** **_Thursday_** every **_2_** months.|
+|`weekly`|- [`interval`][interval link]<br/>- [`days`][days link]<br/>- [`firstDayOfWeek`][firstDayOfWeek link] (optional)|An appointment occurs on *days* every *interval* weeks. Example: An appointment occurs on **_Tuesday_ and _Thursday_** every **_2_** weeks.|
+|`yearly`|- [`interval`][interval link]<br/>- [`dayOfMonth`][dayOfMonth link]<br/>- [`dayOfWeek`][dayOfWeek link]<br/>- [`weekNumber`][weekNumber link]<br/>- [`month`][month link]|1. An appointment occurs on day *dayOfMonth* of *month* every *interval* years. Example: An appointment occurs on day **_7_** of **_September_** every **_4_** years.<br/>2. An appointment occurs on the *weekNumber* *dayOfWeek* of *month* every *interval* years. Example: An appointment occurs on the **_first_** **_Thursday_** of **_September_** every **_2_** years.|
+
+> [!NOTE]
+> If a property is not listed then it is not applicable for that type.
+
+## Set recurrence for a series as the organizer
+
+Along with the recurrence pattern, you also need to determine the start and end dates/times of your appointment series. The [`SeriesTime`][SeriesTime link] object is used to manage that information.
+
+The appointment organizer can set up the appointment series in compose mode only. In the following example, the appointment series is set to occur from 10:30 AM to 11:00 AM PST every Tuesday and Thursday during the period November 2, 2017 to December 2, 2017.
+
+```js
+var seriesTimeObject = new Office.SeriesTime();
+seriesTimeObject.setStartDate(2017,10,2);
+seriesTimeObject.setEndDate(2017,11,2);
+seriesTimeObject.setStartTime(10,30);
+seriesTimeObject.setDuration(30);
+
+var pattern = {
+    "seriesTime": seriesTimeObject,
+    "recurrenceType": "weekly",
+    "recurrenceProperties": {"interval": 1, "days": ["tue", "thu"], "firstDayOfWeek": "sun"},
+    "recurrenceTimeZone": {"name": "Pacific Standard Time"}};
+
+Office.context.mailbox.item.recurrence.setAsync(pattern, callback);
+
+function callback(asyncResult)
+{
+    console.log(JSON.stringify(asyncResult));
+}
+```
+
+> [!NOTE]
```

Comment:
This note seems redundant. The heading already says "as the organizer" and the paragraph explains that the appointment organizer can set up the appointment.